src/locales/it/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}}\nevita l’attacco!",
   "attackHitsCount": "Colpito {{count}} volta/e!",
   "rewardGain": "Ricevi\n{{modifierName}}!",
+  "rewardGainCount": "Ricevi\n{{count}} {{modifierName}}!",
```

```suggestion
  "rewardGainCount": "Ricevi\n{{count}} $t(modifierName, {\"count\": {{count}} })!",
```

For possible plural handling